R/rplot.R

```
@@ -99,4 +99,5 @@ rplot.default <- function(x, y = NULL, ...,
   
   print(r)
   invisible()
```

I'm not sure what role `invisible()` is performing here; if we want to silently return `r`, we should probably wrap it with `invisible()`, no? e.g.

```
return(invisible(r))
```

---

Right. That's confusing.